### ISSUE TITLE
Smartphone app for time lapse photography

### ISSUE THREAD

**Issue by gpereyrairujo** [vuelaendron/vuela#10]
We are currently using a smartphone strapped to our drone as a camera, but we haven't found the best app to capture a series of time-lapse individual images (not video). Ideally, it should be able to control camera settings, such as white balance, ISO, etc. - any suggestions?

Actualmente estamos usando un teléfono atado al dron como cámara, pero no hemos podido identificar la mejor aplicación para capturar una serie de fotografías individuales (no video). Idealmente, debería permitir controlar seteos de la cámara, como balance de blancos, ISO, etc. - sugerencias?

**Comment by lotelx** [comment]
If you want to use an open source app, you can try SkyCam, from PublicLab:
https://publiclab.org/wiki/sky-camera

**Comment by gpereyrairujo** [comment]
It looks great! Very useful functions!!!
I tried it on my phone, and the app closes when it takes a picture :( - maybe it is not compatible with new android versions maybe? have you used it lately?

Se ve bien! Muy útiles las funciones!!!
La probé en mi teléfono, y la app se cierra al momento de tomar una foto :( - quizás no es compatible con versiones nuevas de android? la has usado últimamente?

**Comment by pazbc** [comment]
Yo la probaré ahora 
I'll try it now - we can also ask public lab folks

**Comment by lotelx** [comment]
I just tested now in my phone (Android 7.0) and works well.
But I can not found the place where it saves the photos.
(The guide says Sky Cam Folder) Maybe because the guide says SD storage and my phone don't have any card?

**Comment by pazbc** [comment]
Yeah, same here. Installed it, worked well, and can't find the folder. It probably is the SD card.

**Comment by NicolasNarvaez** [comment]
Hello!, i also tested it, a gallery app found the skycam folder and then i found it in the inner storage (not the sd card storage) on the folder SkyCamFolder, in other words: 
Internal Storage / SkyCamFolder
Hola!, yo también la probé y una aplicación de galería encontró la carpeta, y luego la encontré en el almacenamiento interno (no en la targeta sd), en la carpeta SkyCamFolder, en otras palabras:
Internal Storage / SkyCamFolder
Regards!/Saludos! ;)

**Comment by pazbc** [comment]
@lotelx te quiero mandar ahora un mensaje con más detalles de los planes con Vuela. ¿te lo mando por mensaje en twitter? No pillé tu email

**Comment by gpereyrairujo** [comment]
En el taller del 28 de junio volvimos a probar la app Skycam en 3 teléfonos y en ninguno fucionó :(

Para poder tomar fotos usamos la app OpenCamera (https://play.google.com/store/apps/details?id=net.sourceforge.opencamera&hl=es_AR). 

Se puede configurar un temporizador (para que empiece a tomar fotos unos segundos después de activarla), el intervalo de ráfaga (cada cuántos segundos toma una foto), y la cantidad de fotos que toma. Para este último punto tiene la opción "ilimitado", pero no funcionaba, por lo que usamos la opción "10x" que sólo toma 10 fotos. Sería bueno probar si la opción "ilimitado" funciona bien en otros teléfonos, o es un error de la app.

![screenshot_2018-07-01-16-42-19](https://user-images.githubusercontent.com/8480839/42138143-f6daf7d0-7d4e-11e8-8e98-8f3946ee5beb.png)

![screenshot_2018-07-01-16-40-50](https://user-images.githubusercontent.com/8480839/42138142-ee753696-7d4e-11e8-98d7-2d4cd1b7ba3c.png)

Acá un ejemplo de las fotos que tomamos:

![img_20180629_154630](https://user-images.githubusercontent.com/8480839/42138144-fcbdc8c6-7d4e-11e8-91bd-839fef1b86af.jpg)

Habrá que seguir explorando esta app (u otras), ya que tiene otras funciones interesantes.
